### PR TITLE
[#2392] Polish texts in the ordering wizard

### LINKED ISSUE #2392: Polish texts in open access wizard
There should be more focus on "pinning" the resource to the project. On the buttons and in texts in the wizard.

Tasks:

- [x] (@abacz ) streamline the text content
- [x] Implement changes

image

### PR DESCRIPTION
**Pull request by goreck888**
Remove i18n translations
and main change - last step as Pin to a project
Closes #2392

### CODE REVIEW

File: app/controllers/services/summaries_controller.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def create
 
   private
     def next_title
-      I18n.t("services.summary.#{helpers.map_view_to_order_type(@offer)}.order.title")
+      @offer.orderable? ? "Send access request" : "Pin!"
```
Comment (jswk) [inline app/controllers/services/summaries_controller.rb:30]:
```ruby
@offer.orderable? ? _("Send access request") : _("Pin!")
```
?

**Review by jswk — CHANGES_REQUESTED** [review]

File: app/controllers/services/summaries_controller.rb (hunk 1 above)
Comment (goreck888) [reply app/controllers/services/summaries_controller.rb:30]: Yes

**Comment by jswk** [comment]
But still, tests fail.

**Comment by Marcelinna** [comment]
pr
image
prev
image

**Comment by Marcelinna** [comment]
For orderable change Next - Final details to Next too

**Review by michal-szostak — APPROVED** [review]

**Review by kmarszalek — APPROVED** [review]

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**